docs/4.2/architecture/teleport_proxy.md

```diff
@@ -91,7 +91,7 @@ client `ssh` or using `tsh`:
 
 !!! tip "NOTE": 
     
-    Teleport's proxy command makes it compatible with [SSH jump hosts](https://wiki.gentoo.org/wiki/SSH_jump_host) implemented using OpenSSH's `ProxyCommand`. also supports OpenSSH's ProxyJump/ssh -J implementation as of Teleport 4.1.
+    Teleport's proxy command makes it compatible with [SSH jump hosts](https://wiki.gentoo.org/wiki/SSH_jump_host) implemented using OpenSSH's `ProxyCommand`. Also supports OpenSSH's ProxyJump/ssh -J implementation as of Teleport 4.1.
```

Comment:
I'd change this to "It also" rather than "Also" - I think that's what it was originally before something ate the "it".

Reply:
revised